gtsam/hybrid/tests/testDCMarginals.cpp

```diff
@@ -0,0 +1,41 @@
+/* ----------------------------------------------------------------------------
+
+ * GTSAM Copyright 2010, Georgia Tech Research Corporation,
+ * Atlanta, Georgia 30332-0415
+ * All Rights Reserved
+ * Authors: Frank Dellaert, et al. (see THANKS for the full author list)
+
+ * See LICENSE for the license information
+
+ * -------------------------------------------------------------------------- */
+
+/**
+ *  @file  testDCMarginals.cpp
+ *  @brief Unit tests for DCMarginals class
+ *  @author Varun Agrawal
+ *  @date December 2021
+ */
+
+#include <CppUnitLite/TestHarness.h>
+#include <gtsam/base/serializationTestHelpers.h>
+#include <gtsam/hybrid/DCMarginals.h>
+
+using namespace std;
+using namespace gtsam;
+using namespace serializationTestHelpers;
+
+//******************************************************************************
+TEST(DCMarginals, Constructor) {
+  Marginals continuous;
+  DiscreteMarginals discrete;
+
+  DCMarginals dc_marginals(continuous, discrete);
+  EXPECT(assert_type("gtsam::DCMarginals", dc_marginals));
```

Comment:
I don’t understand. Self-fulfilling prophecy?

Reply:
This is a barebones test for the default constructor. This ensures that everything is included correctly and that a basic operation works.

The idea is that since this is a test, it should assert something, so just assert that the type is as expected.

Reply:
C++ is strongly typed. Test anything, but type is known. Let's cherry pick `assert_type` in a branch if we ever need it, but please leave out of this PR.